python/paddle/fluid/layers/io.py

```diff
@@ -780,6 +792,7 @@ def create_py_reader_by_data(capacity,
        name(basestring): The prefix Python queue name and Reader name. None will
             be generated automatically.
        use_double_buffer(bool): Whether use double buffer or not.
+       prepend(bool): Whether to prepend the reader in front of the main program.
```

Comment:
when should this be true and when should be false?

Reply:
When user wants to prepend `py_reader` in front of the program, it should be `True`, such as programs loaded from the previously saved ones without any read_ops.